datadog/resource_datadog_downtime.go

```diff
@@ -46,7 +46,7 @@ func resourceDatadogDowntime() *schema.Resource {
 				Optional: true,
 				DiffSuppressFunc: func(k, oldVal, newVal string, d *schema.ResourceData) bool {
 					_, startDatePresent := d.GetOk("start_date")
-					return startDatePresent
+					return startDatePresent || (newVal == "0" && oldVal != "0")
```

Comment:
@therve can you add examples when the diff should (not) be supressed?

Reply:
```suggestion
					// "start_date" is set or was previously set (oldVal != "0") and it is being removed (newVal == "0")
					return startDatePresent || (oldVal != "0" && newVal == "0")
```

Reply:
I added a comment, let me know if it makes sense.